recipes/pangeo-dask/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "0.0.2" %}
+
+package:
+  name: pangeo-dask
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask-kubernetes >=0.10
+    - dask >=2.11
+    - distributed >=2.11
+    - tornado >=6
+
+test:
+  imports:
+    - dask
+
+about:
+  home: http://pangeo.io
+  license: MIT
+  license_file: LICENSE
+  summary: "pangeo dask dependencies"
+
+extra:
+  recipe-maintainers:
+    - scottyhq 
```

Comment:
@scottyhq I added only you but we list as many pangeo people as you like here.

Reply:
Fine for now as long as we can add others later (@tjcrone and @jhamman for example)

Reply:
Yep. You can do that later.